zoning/data_processing/eval.py

```diff
@@ -109,23 +131,27 @@ async def evaluate_term(
     search_method: SearchMethod,
     extraction_method: ExtractionMethod,
     k: int,
+    tournament_k: int
 ):
     eval_task = progress.add_task(f"Evaluating {term}", total=len(gt))
 
     # Generate results for the given term in parallel, showing progress along
     # the way.
     results = []
+    row_count = 0
     for row in gt.iter_rows(named=True):
         town = row["town"]
         district = District(full_name=row["district"], short_name=row["district_abb"])
         progress.update(
             eval_task, description=f"Evaluating {term}, {town}, {district.full_name}"
         )
         async for result in compute_eval_result(
-            town, district, term, row, search_method, extraction_method, k
+            town, district, term, row, search_method, extraction_method,
+            k, tournament_k
         ):
             results.append(result)
         progress.advance(eval_task)
+        row_count += 1
```

Comment:
`row_count` should always be the same as `len(gt)`, no? Let's just use that then. That said, I'm not sure we need to save this as a metric, because this should always be the same as `num_eval_rows`, which is an input parameter, right?